pkg/network/vmi/status/updater.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 The KubeVirt Authors.
+ *
+ */
+
+package status
+
+import (
+	"fmt"
+
+	k8scorev1 "k8s.io/api/core/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/namescheme"
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+	"kubevirt.io/kubevirt/pkg/virt-controller/network"
```

Comment:
This is problematic. We should not depend into the core components, it should be the opposite.

Reply:
I am aware of it and plan to take care of it in a follow-up.

It is due to: https://github.com/kubevirt/kubevirt/blob/463dbd6de05985b96b5a28333ad7f8f7e79c925d/pkg/virt-controller/network/multus_annotations.go#L32

Alternatively, I can create a PR to be a prerequisite to this one (and #12816) to move this function under pkg/network/vmispec.

Reply:
Unfortunally, we have it here as well:
https://github.com/kubevirt/kubevirt/blob/eb7d40543cea5175c01f050abfd494f84f5cf29c/pkg/network/pod/annotations/generator.go#L34